_posts/2023-07-20-servo-web-platform-tests.md

```diff
@@ -0,0 +1,83 @@
+---
+layout:     post
+tags:       blog
+title:      Servo and the Web Platform Tests
+date:       2023-07-20
+summary:    How we use the Web Platform Tests to guide Servo’s development.
+categories:
+---
+
+With over 52000 tests and nearly two million subtests, the [Web Platform Tests](https://web-platform-tests.org) are one of the most important parts of the web’s interoperability and compatibility story. Servo has long used the WPT to catch regressions and [flakiness](/blog/2023/03/16/making-easier-to-contribute/), but the suite has also played a role in our [layout engine migration](/blog/2023/04/13/layout-2013-vs-2020/) and guiding our way towards CSS2 conformance.
+
+<style>
+    ._runin {
+        margin-bottom: 1em;
+    }
+    ._runin > p,
+    ._runin > h2 {
+        display: inline;
+    }
+</style>
+
+<div class="_runin"><p>You can find Servo on </p><h2><a href="https://wpt.fyi">wpt.fyi</a></h2><p>, a dashboard for all browsers integrated with the Web Platform Tests.</p></div>
+
+To see pass rates for Servo, click <span style="font-size: 87.5%;">EDIT</span>, <span style="font-size: 87.5%;">ADD PRODUCT</span>, choose Servo, then click <span style="font-size: 87.5%;">SUBMIT</span>.
```

Comment:
Should we just link to Servo results in the previous link and remove this sentence? I mean linking to https://wpt.fyi/results/?label=master&label=experimental&product=servo

Reply:
added a link to that in bc7a5b8cec44ce5a651d03ef401d87e7abf41506, but i kept the sentence, because it’s really not very obvious how to get results for servo if you just type wpt.fyi